simapp/app.go

```diff
@@ -208,7 +207,7 @@ func NewSimApp(
 		authzkeeper.StoreKey,
 	)
 	tkeys := sdk.NewTransientStoreKeys(paramstypes.TStoreKey)
-	memKeys := sdk.NewMemoryStoreKeys(capabilitytypes.MemStoreKey)
+	memKeys := sdk.NewMemoryStoreKeys(capabilitytypes.MemStoreKey, "testing")
```

Comment:
I'd make "testing" be a variable testingKey or maybe rename to `testingkey`. It isn't immediately clear what the "testing" argument is used for without looking up the API of `NewMemoryStoreKeys`

I'm a little worried chains might blindly copy these changes without an explicit comment telling them not to

Reply:
Just changed it to `testingkey` and added comment